Constructor api test reports

susansjy22 opened the pull request:
Updated reports to work within constructor API branch

Commit:
dummy

myushen committed:
update functions for anndata input work

myushen committed:
update module for anndata/sce input

myushen committed:
merge, solve conflicts from master

Merge branch 'master' into module-data-input

Conflicts:
	R/functions.R
	R/modules_grammar_hpc.R

myushen committed:
fix

myushen committed:
fix package dependency

myushen committed:
fix mt-ribsome gene identification and pipeline running

Commit:
update emptydroplets

Commit:
merge upstream

Commit:
fix conflicts

Merge remote-tracking branch 'upstream/master' into modules_grammar_HPC

Conflicts:
	DESCRIPTION
	R/execute_pipeline.R
	R/utilities.R
	README.rmd
	man/cell_cycle_scoring.Rd
	man/doublet_identification.Rd

Commit:
Merge remote-tracking branch 'upstream/master' into modules_grammar_HPC

myushen committed:
fix

myushen committed:
merge from master

Merge branch 'master' into module-data-input

Conflicts:
	tests/testthat/test_single_functions.R

Commit:
Merge remote-tracking branch 'upstream/master' into modules_grammar_HPC

Merge updates for sample_col
:wq

Commit:
update testing

Commit:
clean readme

Commit:
update run_target_pipeline() steps

myushen committed:
fix conflicts

Merge remote-tracking branch 'remotes/upstream/custom_input_names' into transform-assays

Conflicts:
	R/functions.R
	R/modules_grammar_hpc.R
	tests/testthat/test_single_functions.R

myushen committed:
save transform assay in different formats

myushen committed:
fix pipeline

Commit:
update testing fibrosis

Merge remote-tracking branch 'upstream/module-data-input' into modules_grammar_HPC

Conflicts:
	R/functions.R
	R/utilities.R

myushen committed:
fix the pipeline for census anndata input

myushen committed:
testthat for census data

myushen committed:
update transformation function

myushen committed:
update transformation function

myushen committed:
empty_droplet based on custom filter threshold

myushen committed:
Merge branch 'custom_input_names' into transform-assays

myushen committed:
update azimuth reference

myushen committed:
Merge branch 'azimuth_annotation' into transform-assays

myushen committed:
test pilot data

myushen committed:
solve conflicts

Merge branch 'transform-assays' of github.com:myushen/HPCell into transform-assays

# Conflicts:
#	R/modules_grammar_hpc.R
#	R/utilities.R
#	man/empty_droplet_threshold.Rd
#	tests/testthat/test-census-samples.R

myushen committed:
update azimuth reference

stemangiola committed:
Merge pull request #78 from MangiolaLaboratory/constructor-API

Constructor api

stemangiola committed:
Update DESCRIPTION

stemangiola committed:
Update DESCRIPTION

myushen committed:
update azimuth

stemangiola committed:
track files

stemangiola committed:
version UP

myushen committed:
track files and solve conflicts.

Merge branch 'master' into transform-assays

# Conflicts:
#	R/factories.R
#	R/functions.R
#	R/modules_grammar_hpc.R
#	R/tranform_assay.R
#	R/utilities.R
#	man/test_differential_abundance.Rd
#	man/transform_utility.Rd
#	tests/testthat/test_single_functions.R

stemangiola committed:
fix transform assay

stemangiola committed:
add source argument to sutom functions

stemangiola committed:
deal with undefined targets

stemangiola committed:
deal with undefined targets

stemangiola committed:
fixed bug

stemangiola committed:
fixed bug

stemangiola committed:
still a bug with SCT filtering

stemangiola committed:
still a bug with SCT filtering

stemangiola committed:
clean backend _target.R file

myushen committed:
fix transform_assay

Commit:
Update reports

Commit:
transform assays

Commit:
fix merge conflicts from construction-API

Merge remote-tracking branch 'upstream/constructor-API' into constructor-API-test-reports

# Conflicts:
#	R/factories.R
#	R/modules_grammar_hpc.R
#	README.rmd
#	man/test_differential_abundance.Rd
#	tests/testthat/test_single_functions.R

Commit:
loadPBMC

myushen committed:
update testthat

myushen committed:
fix fetching blueprint reference

myushen committed:
update testthat

stemangiola committed:
update consensus function

stemangiola committed:
complete the consensus function

stemangiola committed:
improve the consensus function

stemangiola committed:
move cellChat to dev because it is using the version v1 rather than v2, we need to update

stemangiola committed:
drop cellchat dependencies

stemangiola committed:
fix consensus function

stemangiola committed:
update exports

stemangiola committed:
pull

Merge branch 'master' of github.com:Melbourne-COVID-Predict/jascap

# Conflicts:
#	NAMESPACE

Commit:
Update report generation and testing

stemangiola committed:
mainly update cell consensus

stemangiola committed:
Merge branch 'drop-undefined-datasets' of https://github.com/MangiolaLaboratory/HPCell into drop-undefined-datasets

myushen committed:
update pipeline

stemangiola committed:
Merge pull request #81 from MangiolaLaboratory/drop-undefined-datasets

Drop undefined datasets

myushen committed:
merge master and solve conflicts

Merge remote-tracking branch 'origin/master' into transform-assays

# Conflicts:
#	NAMESPACE
#	R/CellChat.R
#	R/functions.R
#	R/modules_grammar_hpc.R
#	R/tranform_assay.R
#	R/utilities.R
#	tests/testthat/test_single_functions.R

stemangiola committed:
Merge pull request #76 from myushen/transform-assays

Transform assays

stemangiola committed:
Merge pull request #82 from MangiolaLaboratory/custom_input_names

Custom input names

stemangiola committed:
fix pipeline

stemangiola committed:
add man

stemangiola committed:
add man

stemangiola committed:
pull

Merge branch 'master' of github.com:Melbourne-COVID-Predict/jascap

# Conflicts:
#	R/modules_grammar_hpc.R
#	R/tranform_assay.R

Commit:
update reports

stemangiola committed:
update pipeline to work with HCA

stemangiola committed:
update functions

myushen committed:
fix transform calculation

myushen committed:
fix

stemangiola committed:
Merge pull request #83 from myushen/master

fix transform calculation

stemangiola committed:
fix temporary fix

stemangiola committed:
typo

stemangiola committed:
Merge branch 'master' of github.com:Melbourne-COVID-Predict/jascap

myushen committed:
rename assay names for consistency

stemangiola committed:
add idle time < inf

stemangiola committed:
do not skip if empty is null

stemangiola committed:
typo

stemangiola committed:
move step

stemangiola committed:
drop this filter, it does not make sense. The next steps can be robust

stemangiola committed:
Merge pull request #84 from myushen/master

rename assay names for consistency

Commit:
report tests passed

myushen committed:
fix estimated counts in transform function

myushen committed:
Merge branch 'master' of github.com:myushen/HPCell

myushen committed:
fix

myushen committed:
round estimated counts

myushen committed:
metacell module prototype

Commit:
functional reports

stemangiola commented:
Amazing, please post here the code to reproduce the reports, please test the code before, so I can reproduce with copy and paste.

Commit:
commit updated reports

stemangiola commented:
Thanks , could you please have a look at the github render, it is strange. Also could you replicate using a dataset from 

```r
scRNAseq::fetchDataset("baron-pancreas-2016", "2023-12-14", path="human") 
```

you code should not include any absolute path, as that is not reproducible.

Commit:
updated reports FINAL

bhuvad committed:
remove duplicate package imports

william-hutchison committed:
Fix incorrect function imports

william-hutchison committed:
Fix incorrect remote address

william-hutchison committed:
Version up

william-hutchison committed:
Fix typo in code snippet

william-hutchison committed:
Version up

william-hutchison committed:
Add SeuratDisk remote

william-hutchison committed:
Add NAMESPACE imports to DESCRIPTION

william-hutchison committed:
Update examples

william-hutchison committed:
Version up

stemangiola committed:
Merge pull request #87 from william-hutchison/fix-installation

Fix installation

stemangiola committed:
clean the object in transform assay. and check if min counts are > 0

stemangiola committed:
Merge branch 'master' of github.com:Melbourne-COVID-Predict/jascap

myushen committed:
Merge branch 'MangiolaLaboratory:master' into master

myushen committed:
update

william-hutchison committed:
Fix incorrect variables passed to functions

william-hutchison committed:
Create SCT assay before attempting assignment

william-hutchison committed:
Prevent error when cell annotations are unavailable

william-hutchison committed:
Remove NaN features from SCT assay

william-hutchison committed:
Version up

william-hutchison committed:
Remove undefined function export

william-hutchison committed:
Version up

stemangiola committed:
Stefano's changes made directly on Github

myushen committed:
fix transform majority counts in estimation

myushen committed:
split metacell function

bhuvad committed:
Added code to compute the mapping and added data required for mapping.

myushen committed:
solve single-column assay in sce object

myushen committed:
metacell preprocessing, clustering and module update

myushen committed:
Merge pull request #85 from myushen/master

fix transform function

stemangiola committed:
Merge pull request #89 from william-hutchison/fix-SCTransform

Fix SCTransform

stemangiola committed:
Update utilities.R

myushen committed:
fix transform for missing cells issue

myushen committed:
hpcell pseudobulk module accept saving to H5ad

myushen committed:
Merge branch 'consensus_celltyping'
Merging consensus cell typing from Dharmesh for fetching curated cell
type

myushen committed:
cell type concensus module and pseudobulk

myushen committed:
preprocess_simplify as a function not a module

myushen committed:
fix

myushen committed:
pseudobulk and concensus module dependency

myushen committed:
metacell functions and unit tests

myushen committed:
replace NAs in annotations tibble to save anndata correctly

myushen committed:
metacell module

myushen committed:
Merge branch 'MangiolaLaboratory:master' into metacell

myushen committed:
Merge branch 'metacell' of github.com:myushen/HPCell into metacell

myushen committed:
Merge branch 'metacell' of github.com:myushen/HPCell into metacell

myushen committed:
Merge branch 'metacell' into master

myushen committed:
Merge pull request #2 from myushen/master

merge my master

myushen committed:
Merge branch 'metacell' of github.com:myushen/HPCell into metacell

myushen committed:
merge the lastest cell-type-concensus function

Merge branch 'pseudobulk_module' into metacell

# Conflicts:
#	R/modules_grammar_hpc.R

myushen committed:
make metacell module work

myushen committed:
fix metacell function error for large pipeline

myushen committed:
argument for users to choose mininum cells in metacell

myushen committed:
fix metacell module

myushen committed:
gamma documentation

myushen committed:
fix alive identification module

myushen committed:
cell chat module

myushen committed:
append sample_id to cellchat tbl

myushen committed:
Merge branch 'master' into cellchat
merge alive_identification from master

myushen committed:
fix

myushen committed:
update doublet_identification function and metacell dependencies

myushen committed:
fix

myushen committed:
remove alive_tbl dependency in doublet function. force scDblFinder() work for small number of cells.

myushen committed:
mark fail scdblfinder sample doublet to unknown

myushen committed:
Merge branch 'master' into cellchat
merge doublet module in master to cellchat

Commit:
Update report formatting

Commit:
add new reports remove old ones

myushen committed:
skip cellchat object with only one celltype

Commit:
delet old report rmd files

Commit:
Add empty droplet report qmd file

myushen committed:
solve incorrect imported function

Commit:
change to use tar_quarto_raw(), and execute_param

Commit:
pull from upstream

Merge branch 'constructor-API-test-reports' of github.com:susansjy22/HPCell into constructor-API-test-reports
# Please enter a commit message to explain why this merge is necessary,
# especially if it merges an updated upstream into a topic branch.
#
# Lines starting with '#' will be ignored, and an empty message aborts
# the commit.

myushen committed:
universal cell type unified function, empty droplet percentage for rhasopdy tech

myushen committed:
adjust knn, npc for small number of cells in a sample

myushen committed:
redesign metacell for little sample cells. marks less than 3 single cells in a sample to metacell_2 = 1

myushen committed:
add communication weights/strength

myushen committed:
add pathway strength. remove thresh 0.05, return all communication, both insignificant and significant

myushen committed:
make feature_thresh as an input target in remove_empty_threshold module

myushen committed:
merge cellchat to master

Merge branch 'cellchat'

# Conflicts:
#	R/functions.R

myushen committed:
fix and cellchat mouse option

susansjy22 committed:
dummy

susansjy22 committed:
pull updates from myushen/master

Merge remote-tracking branch 'myushen/master' into constructor-API-test-reports

# Conflicts:
#	Pipeline_benchmarking_script.R
#	R/data.R
#	R/execute_pipeline.R
#	R/factories.R
#	R/functions.R
#	R/modules_grammar_hpc.R
#	R/utilities.R
#	man/alive_identification.Rd
#	man/annotation_label_transfer.Rd
#	man/create_pseudobulk.Rd
#	man/doublet_identification.Rd
#	man/empty_droplet_id.Rd
#	man/test_differential_abundance.Rd
#	tests/testthat/test_single_functions.R

susansjy22 committed:
pull changes from remote

Merge branch 'constructor-API-test-reports' of github.com:susansjy22/HPCell into constructor-API-test-reports

# Conflicts:
#	NAMESPACE
#	R/CellChat.R

susansjy22 committed:
Update Utilities

susansjy22 committed:
Fix pseudobulk to save the output

susansjy22 committed:
Remove commented function

susansjy22 committed:
Remove commented function

susansjy22 committed:
Add ensembl_genes_biomart.rda
